src/main/java/io/confluent/connect/jdbc/sink/metadata/SchemaPair.java

```diff
@@ -46,4 +46,8 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash(keySchema, valueSchema);
   }
+
+  public String toString() {
+    return String.format("<SchemaPair: %s, %s>", keySchema.toString(), valueSchema.toString());
```

Comment:
Since the varargs parameter of `String.format(...)` is of type `Object` for which `toString()` will be called if non-null, it's a bit safer to do the following in case the key schema or value schema are null:
```suggestion
    return String.format("<SchemaPair: %s, %s>", keySchema, valueSchema);
```